docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+# As of 9-12-17, Amazon ECS CLI supports Docker compose file syntax versions
+# 1 and 2
+# (see: http://docs.aws.amazon.com/AmazonECS/latest/developerguide/cmd-ecs-cli-compose.html)
+version: '2'
```

Comment:
It is great to have this file to tie everything together for us. I find this much easier to deal with than the previous bash scripts. Thank you for pulling it together, @rzelayafavila!